amenity-points.mss

```diff
@@ -2499,6 +2560,10 @@
     [feature = 'amenity_shelter'] {
       text-fill: @amenity-brown;
     }
+    [access != ''][access != 'permissive'][access != 'yes'] {
+      text-opacity: 0.33;
+      text-halo-radius: 0;
+    }
```

Comment:
This accidentally changes the text to be semi-transparent for all the above features if they have a value of `access=` other than `yes` or `permissive` - not just for amenity=shelter but also for tourism=hotel/motel/hostel/chalet/ etc

Reply:
This line was already fixed in #3616, sorry